org.bridgedb.ws.core/test/org/bridgedb/ws/core/TestCapabilitiesBean.java

```diff
@@ -0,0 +1,316 @@
+package org.bridgedb.ws.core;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.bridgedb.BridgeDb;
+import org.bridgedb.DataSource;
+import org.bridgedb.IDMapper;
+import org.bridgedb.IDMapperException;
+import org.bridgedb.Xref;
+import org.bridgedb.ws.bean.CapabilitiesBean;
+import org.bridgedb.ws.bean.DataSourceBean;
+import org.bridgedb.ws.bean.DataSourceMapBean;
+import org.bridgedb.ws.bean.FreeSearchSupportedBean;
+import org.bridgedb.ws.bean.PropertyBean;
+import org.bridgedb.ws.bean.XrefBean;
+import org.bridgedb.ws.bean.XrefMapBean;
+import org.bridgedb.ws.bean.XrefsBean;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestCapabilitiesBean {
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@Test
+	public void testCapabilitiesBean() throws ClassNotFoundException, IDMapperException {
```

Comment:
This is a very long method for a **unit** test. Maybe split into smaller methods?